src/AmqpQueue.php

```diff
@@ -37,7 +37,7 @@ public function setArguments(array $arguments);
 
     /**
      * @param string $key
-     * @param string|bool|int|float $value
+     * @param mixed  $value
```

Comment:
So do we want to list all php types except objects and resources (and thus include `null` and `array`)?